documentation/modules/adding-source-provider.adoc

```diff
@@ -78,6 +80,7 @@ ifdef::vmware[]
 . Specify the following fields:
 
 * *Provider resource name*: Name of the source provider.
+* *Endpoint type*: Select the vSphere provider endpoint type. Options: *vSphere* or *ESXi*.
```

Comment:
it's vCenter or ESXi , @ahadas please keep me honest here

Reply:
Fixed.